pkg/controller/status/status.go

```diff
@@ -300,6 +310,21 @@ func (c *Controller) merge(existing *configv1.ClusterOperator) *configv1.Cluster
 		} else {
 			removeOperatorStatusCondition(&existing.Status.Conditions, InsightsDownloadDegraded)
 		}
+		if len(ocmErrorMsg) > 0 {
+			setOperatorStatusCondition(&existing.Status.Conditions, configv1.ClusterOperatorStatusCondition{
+				Type:               configv1.OperatorDegraded,
+				Status:             configv1.ConditionTrue,
+				LastTransitionTime: metav1.Time{Time: last},
+				Reason:             ocmErrorReason,
+				Message:            ocmErrorMsg,
+			})
+		} else {
+			setOperatorStatusCondition(&existing.Status.Conditions, configv1.ClusterOperatorStatusCondition{
+				Type:   configv1.OperatorDegraded,
+				Status: configv1.ConditionFalse,
+				Reason: "AsExpected",
+			})
+		}
```

Comment:
Hmmm this block is probably not necessary and makes our tests failing... need to check